### PR TITLE
Improve navigation; highlight keynotes

### PR DESCRIPTION
**Pull request by morgantaschuk**
- Fixed the 'Partners' page so that sponsors are first and other partners are second; fixed the dead link on that page
- Moved the 'Promote' nav bar item under the 'About' section
- Prominently add Keynote speakers to the front page and schedule page, and add their bios
- Moved the 'Training' nav bar item to the 'Schedule section'
- Changed the text on the Submissions page to state that abstract submissions are closed
- Other tweaks and fixups

### CODE REVIEW

File: _posts/2020-02-11-training-vote.markdown [hunk 1]
```diff
@@ -7,7 +7,7 @@ image: hero.jpg
 color: olive
 ---
 
-**[Vote for the Training Topics you want!](/training/)**
+**Vote for the Training Topics you want!)**
```
Comment (heuermh) [inline _posts/2020-02-11-training-vote.markdown]: Missed removing `)` here

**Review by heuermh — CHANGES_REQUESTED** [review]
LGTM, thank you, @morgantaschuk!  Found one typo, see review comment

File: _posts/2020-02-11-training-vote.markdown (hunk 1 above)
Comment (morgantaschuk) [reply _posts/2020-02-11-training-vote.markdown]: Thanks! Fixed.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,12 @@
...
-
-
-### Quick-start guide
-1. [Fork](https://github.com/bcc2020/bcc2020.github.io) this repo
-2. Clone locally
-3. Update ```_config.yml```
-4. Select what content blocks do you need
-5. Push changes to ```gh-pages``` branch
-6. Enjoy your awesome site at ```http://[your github name].github.io/```
+Template is brought by [GDG Lviv](http://lviv.gdg.org.ua/) team. Substantial modifications were made to get it working for a non-GDG event.
```
Comment (denis-yuen) [inline README.md]: Link doesn't work or is down

**Review by denis-yuen — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (morgantaschuk) [reply README.md]: Fixed

**Review by tnabtaf — APPROVED** [review]

**Pull request merged**